.github/.settings.yml

```diff
@@ -0,0 +1,300 @@
+    color: ededed
+  - name: PM prio 1
+    color: fbca04
+  - name: PM prio 2
+    color: fbca04
+  - name: PM prio 3
+    color: fbca04
+  - name: QA
+    color: 1d76db
+  - name: QA-blocker
+    color: 0052cc
+  - name: QA-known
+    color: 0052cc
+  - name: QA-ready
+    color: 0052cc
+  - name: QA-team
+    color: 4cc948
+  - name: QA-testing
+    color: 0052cc
+  - name: Sys Admin
+    color: ededed
+  - name: WIP
+    color: 8c8cea
+  - name: Waiting
+    color: e6e6e6
+  - name: Waiting Internal
+    color: e6e6e8
+  - name: Waiting On Review/Feedback
+    color: ededed
+  - name: app:dav
+    color: 02d7e1
+  - name: app:encryption
+    color: 02d7e1
+  - name: app:federation
+    color: 02d7e1
+  - name: app:files
+    color: 02d7e1
+  - name: app:files_encryption
+    color: 02d7e1
+  - name: app:files_external
+    color: 02d7e1
+  - name: app:files_trashbin
+    color: 02d7e1
+  - name: app:files_versions
+    color: 02d7e1
+  - name: app:objectstore
+    color: 02d7e1
+  - name: app:provisioning_api
+    color: 02d7e1
+  - name: app:search
+    color: 02d7e1
+  - name: app:user_ldap
+    color: 02d7e1
+  - name: app:windows_network_drive
+    color: ededed
+  - name: backport-request
+    color: 006b75
+  - name: blue-ticket
+    color: 0033CC
+  - name: bounty
+    color: 129e5e
+  - name: browser:chrome
+    color: 207de5
+  - name: browser:edge
+    color: 207de5
+  - name: browser:firefox
+    color: 207de5
+  - name: browser:ie
+    color: 207de5
+  - name: browser:safari
+    color: 207de5
+  - name: bug
+    color: fc2929
+  - name: bug-analysis
+    color: fbca04
+  - name: bug-solved
+    color: 0e8a16
+  - name: clarification request
+    color: d4c5f9
+  - name: comp:apps
+    color: d4c5f9
+  - name: comp:authentication
+    color: d4c5f9
+  - name: comp:caldav
+    color: d4c5f9
+  - name: comp:carddav
+    color: d4c5f9
+  - name: comp:console-occ
+    color: d4c5f9
+  - name: comp:core
+    color: d4c5f9
+  - name: comp:core-js
+    color: d4c5f9
+  - name: comp:filesystem
+    color: d4c5f9
+  - name: core-bug
+    color: ededed
+  - name: credits
+    color: e58729
+  - name: db:mariadb
+    color: '444444'
+  - name: db:mysql
+    color: '444444'
+  - name: db:oracle
+    color: '444444'
+  - name: db:postgresql
+    color: '444444'
+  - name: db:sqlite
+    color: '444444'
+  - name: dependencies
+    color: 0025ff
+  - name: design
+    color: cc317c
+  - name: dev:acceptance-tests
+    color: ff9966
+  - name: dev:appframework
+    color: FF9966
+  - name: dev:integration-tests
+    color: FF9966
+  - name: dev:js-tests
+    color: FF9966
+  - name: dev:ocp
+    color: FF9966
+  - name: dev:ocs
+    color: FF9966
+  - name: dev:phpunit-tests
+    color: FF9966
+  - name: dev:smashbox-tests
+    color: FF9966
+  - name: discussion
+    color: fbca04
+  - name: documentation
+    color: BFDADC
+  - name: enhancement
+    color: 02e10c
+  - name: enhancement-PM
+    color: FF9900
+  - name: feature:activity
```

Comment:
:gun:  all feature:

Reply:
Hm, I'd love to have some `feature:…` labels to categorize issues. What would be the alternative?

Reply:
I'm happy to add feature labels once we have some - as of today everything is files from my understansing. Furthermore the feature labels in core are necessary to manage the masses of thousands of issue which have been opened in there for various of apps.